full-stack-tests/backend/src/integration/CloudSqlite.test.ts

```diff
@@ -188,6 +191,90 @@ describe("CloudSqlite", () => {
     container.disconnect({detach: true});
   });
 
+  it("Should a single user upload changes successfully even if write lock expires", async () => {
```

Comment:
Maybe this should be "(it) should upload a single user's changes successfully when a write lock is expired"

Reply:
resolved in [commit 7ad75934bda38b7d23a0730443088800a94e3716](https://github.com/iTwin/itwinjs-core/pull/7000/commits/7ad75934bda38b7d23a0730443088800a94e3716)